web/modules/custom/farm_nfa/src/Form/ForestPlanGfwForm.php

```diff
@@ -11,6 +11,12 @@
  * @ingroup farm_nfa
  */
 class ForestPlanGfwForm extends FormBase {
+  
+  protected $routeMatch;
+
+  public function __construct() {
```

Comment:
This is not how traditionally DI works, please look for examples in the existing code and the books :)